src/cursor.c

```diff
@@ -342,8 +343,15 @@ cursor_button(struct wl_listener *listener, void *data)
 	desktop_focus_view(&server->seat, view);
 	damage_all_outputs(server);
 
-	if (is_double_click(500) && view_area == LAB_SSD_PART_TITLEBAR) {
-		view_toggle_maximize(view);
+	if (is_double_click(rc.doubleclick_time) && view_area == LAB_SSD_PART_TITLEBAR) {
```

Comment:
We can do this in a better, but let cross that bridge after we’ve got this merged

Reply:
Agreed. I just wanted to get one use-case going (doubleclick the title bar) to test the xml parsing. This will need to change a lot to support more mousebinds cleanly